### PR TITLE
feat: improve Consumer module

### LINKED ISSUE #1630: improve Consumer module
Here are some minor issues on the Consumer page we could resolve.

## TODO

- [ ] search area, using Table's own, using fuzzy search.
- [ ] List page title section, add description: Consumer (Consumer) is XXX.
- [ ] Create page, title: Create Consumer, not `Create Consumer`.
- [ ] Create page, step one, name input box, Placeholder remove spaces.
- [ ] Create page, step one, description under Name input box: consumer name needs to be globally unique; 
- [ ] Create/edit/delete successfully prompt, do not include spaces.
- [ ] list page, add enabled plugin fields, wrap them in tags for display.
- [ ] Edit mode, the title needs to be changed to: Edit Consumer.

### PR DESCRIPTION
**Pull request by Si-ege**
Please answer these questions before submitting a pull request, **or your PR will get closed**.

**Why submit this pull request?**

- [ ] Bugfix
- [x] New feature provided
- [ ] Improve performance
- [ ] Backport patches

**What changes will this PR take into?**

Please update this section with detailed description.

**Related issues**

resolve #1630
1.Create page, add description under name input box 
2.Create/Edit/Delete Success prompt to delete spaces 
3.List page, add enabled plugin fields

**Checklist:**

- [x] Did you explain what problem does this PR solve? Or what new features have been added?
- [x] Have you added corresponding test cases?
- [ ] Have you modified the corresponding document?
- [x] Is this PR backward compatible? If it is not backward compatible, please discuss on the mailing list first

### CODE REVIEW

File: web/src/pages/Consumer/Create.tsx [hunk 1]
```diff
@@ -64,7 +64,7 @@ const Page: React.FC = (props) => {
             username
               ? formatMessage({ id: 'component.global.edit' })
               : formatMessage({ id: 'component.global.create' })
-          } ${formatMessage({ id: 'menu.consumer' })} ${formatMessage({
+          }${formatMessage({ id: 'menu.consumer' })}${formatMessage({
```
Comment (guoqqqi) [inline web/src/pages/Consumer/Create.tsx:67]: Please check if this is displayed correctly in English

**Comment by Si-ege** [comment]
已改 请查阅
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "apache/apisix-dashboard"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2022年2月18日(星期五) 中午1:49
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [apache/apisix-dashboard] feat: improve Consumer module (PR #2327)





 
@guoqqqi commented on this pull request.
 
 
In web/src/pages/Consumer/Create.tsx:
 &gt; @@ -64,7 +64,7 @@ const Page: React.FC = (props) =&gt; {              username                ? formatMessage({ id: 'component.global.edit' })                : formatMessage({ id: 'component.global.create' }) -          } ${formatMessage({ id: 'menu.consumer' })} ${formatMessage({ +          }${formatMessage({ id: 'menu.consumer' })}${formatMessage({  
Please check if this is displayed correctly in English
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by guoqqqi** [comment]
Front-end test cases need to be updated

File: web/cypress/integration/consumer/create_and_delete_consumer.spec.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ context('Create and Delete Consumer', () => {
     monacoScroll: '.monaco-scrollable-element',
     monacoViewZones: '.view-zones',
     notificationCloseIcon: '.ant-notification-close-icon',
+    DetectionofPluginfields: '.ant-tabel-cell',
```
Comment (guoqqqi) [inline web/cypress/integration/consumer/create_and_delete_consumer.spec.js:33]: It doesn't look like it's being used, please remove it

File: web/src/pages/Consumer/List.tsx [hunk 3]
```diff
@@ -64,6 +64,12 @@ const Page: React.FC = () => {
       hideInSearch: true,
       render: (text) => timestampToLocaleString(text as number),
     },
+    {
+      title: formatMessage({ id: 'menu.plugin' }),
+      dataIndex: 'plugins',
+      hideInSearch: true,
+      render: (_, record) => <>{Object.keys(record.plugins).join(',')}</>,
```
Comment (guoqqqi) [inline web/src/pages/Consumer/List.tsx:71]:
```suggestion
      render: (_, record) => Object.keys(record.plugins).join(','),
```

**Comment by guoqqqi** [comment]
also cc @bzp2010 Please help trigger CI ～

**Review by guoqqqi — APPROVED** [review]

**Review by Baoyuantop — APPROVED** [review]

**Review by bzp2010 — APPROVED** [review]

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**